extensions/cornerstone-3d/src/initWADOImageLoader.js

```diff
@@ -13,7 +13,7 @@ function initWebWorkers(appConfig) {
   const config = {
     maxWebWorkers: Math.min(
       Math.max(navigator.hardwareConcurrency - 1, 1),
-      appConfig.maxNumberOfWebWorkers
+      appConfig.maxNumberOfWebWorkers || 3
```

Comment:
This prevents a windows crash.

Reply:
This always return the first value if `appConfig.maxNumberOfWebWorkers` is not 0, which is not in the default.js

Maybe I'm missing the point here

Reply:
It returns:
This defaults maxNumberOfWebWorkers to 3, and then returns the smaller of:
hardwareConcurrency -1, maxNumberOfWebworkers
So in a default situation where max number of webworkers isnt set, and hardware concurrency is high, it will return 3.
On a phone/tablet, it will probably return hardware concurrency (probably 1 or 2).
This prevents a browser crash on Chrome on windows.

Reply:
but if it is set and it is let's say 10, 10||3 is 10, but don't you want it to be 3?